wiki/Tournaments/LGA/2024/en.md

```diff
@@ -0,0 +1,185 @@
+4. Participants will be briefed on further steps via an announcement message on the website.
+5. Tournament staff members are **not** allowed to play in the tournament, with the exception of commentators, statisticians, and streamers.
+   - The tournament managers must ensure that such staff members do not have access to priviledged information, e.g. Qualifiers score information, mappool information before the mappool showcase, etc. They are to be treated as regular players whenever possible.
+
+### Qualifier information
+
+1. In the Qualifier stage, all players will play a specific pool designed by the mappool selectors.
+2. The mappool will contain 4 brackets: No Mod, [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), and [Double Time](/wiki/Gameplay/Game_modifier/Double_Time).
+3. The Qualifier pool will feature 11 beatmaps, organised as such:
+   - No Mod: 4 beatmaps
+   - Hidden: 2 beatmaps
+   - Hard Rock: 2 beatmaps
+   - Double Time: 3 beatmaps
+4. Players will be asked to play the mappool twice at a designated time. The best play for each map will be used for seeding.
+5. The mappool is to be played according to the order listed on this page.
+6. Only the top 32 seeded players will advance to the Bracket stage.
+
+### Bracket stage information
+
+1. Matches for this stage of the tournament will have players play a fixed amount of maps, and the cumulative score difference across all maps will decide the winner.
+2. Draft order will follow the ABBA format, where the first player (A) and the second player (B) will take turns picking and banning beatmaps.
+3. Following the Qualifiers, the bracket stage will have two phases (one per weekend):
+   - For the first weekend, a single-elimination bracket will be played from the Round of 32 up until the Quarterfinals.
+     - All matches in this phase will utilise the same mappool. The first player to score 3 points wins (best of 5). 
+     - The first weekend's pool will feature 11 maps, organised as such:
+     - No Mod: 4 beatmaps
+     - Hidden: 2 beatmaps
+     - Hard Rock: 2 beatmaps
+     - Double Time: 3 beatmaps
+    - The draft order will be as follows:
+      1. The first player (A) will ban one beatmap, followed by the second player (B) also banning a beatmap: AB
+      2. Players will pick two beatmaps respecting the following order: BAAB
+      3. Both players ban will two maps, as such: ABBA
+      4. The last beatmap remaining in the pool will be used as the 5th pick for the match.
+   - For the second weekend, The 8 remaining players from the previous phase will then then play another set of matches, after which the remaining 4 players will be placed on a double-elimination bracket.
+     - All matches in this phase will utilise the same mappool, except for the Losers bracket Finals and Grand Finals, where a new osu! original will be introduced into the pool. The first player to score 4 points wins (best of 7).
```

Comment:
probably like so? i've generally seen two more common ways of writing it and i'm pretty sure both include the apostrophe. going by https://en.wikipedia.org/wiki/Double-elimination_tournament

```suggestion
     - All matches in this phase will utilise the same mappool, except for the Losers' Bracket Finals and Grand Finals, where a new [osu! original](/wiki/osu!_originals) will be introduced into the pool. The first player to score 4 points wins (best of 7).
```